src/databricks/sql/cloudfetch/download_manager.py

```diff
@@ -12,50 +12,90 @@
 
 @dataclass
 class DownloadableResultSettings:
+    """
+    Class for settings common to each download handler.
+
+    Attributes:
+        is_lz4_compressed (bool): Whether file is expected to be lz4 compressed.
+        link_expiry_buffer (int): Time in seconds to prevent download of a link before it expires. Default 0 secs.
+        download_timeout (int): Timeout for download requests. Default 60 secs.
+        max_consecutive_file_download_retries (int): Number of consecutive download retries before shutting down.
+    """
+
     is_lz4_compressed: bool
-    link_expiry_buffer: float = 0.0
-    download_timeout: float = 0.0
+    link_expiry_buffer: int = 0
```

Comment:
are the expiry buffer and timeout in minutes, seconds?

Reply:
Seconds - I'll make this clearer